### PR TITLE
*: remove returned value `isNull` in `Row` methods.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@winkyao @zz-jason PTAL

**Comment by shenli** [comment]
Why should we do this?

**Comment by coocood** [comment]
It's redundant, we have `IsNull` method already.

File: types/row.go [hunk 1]
```diff
@@ -21,40 +21,40 @@ type Row interface {
 	Len() int
 
 	// GetInt64 returns the int64 value and isNull with the colIdx.
```
Comment (lamxTyler) [inline types/row.go]: Update the comment?

File: util/chunk/chunk.go [hunk 2]
```diff
@@ -328,158 +328,144 @@ func (r Row) Len() int {
 }
 
 // GetInt64 returns the int64 value and isNull with the colIdx.
```
Comment (lamxTyler) [inline util/chunk/chunk.go]: Ditto.

**Comment by coocood** [comment]
@lamxTyler PTAL

**Pull request merged**